src/PrestaShopBundle/Form/Admin/Sell/Product/Specification/SpecificationsType.php

```diff
@@ -47,7 +47,6 @@ class SpecificationsType extends TranslatorAwareType
     /**
      * @param TranslatorInterface $translator
      * @param array $locales
-     * @param FormChoiceProviderInterface $productConditionChoiceProvider
```

Comment:
So that I learn I do not understand why you delete this? it's possible when you test `DisablingSwitchType `

Reply:
Mistake! Tried separating bith condition fields into dedicated type, but changed my mind and forgot this change. Thanks for note, will revert